specifyweb/frontend/js_src/lib/components/FormCommands/ShowTransactions.tsx

```diff
@@ -114,10 +116,12 @@ export function ShowLoansCommand({
   return typeof data === 'object' ? (
     <Dialog
       buttons={commonText.close()}
-      header={commonText.transactions()}
+      header={interactionsText.interactions()}
+      icon={<span className="text-blue-500">{icons.chat}</span>}
       onClose={handleClose}
     >
-      <H3>
+      <H3 className={`flex items-center gap-2`}>
```

Comment:
The curly braces here are not needed: a simple string will do!
Curly braces are only needed in an element's className if an expression is needed to be evaluated. 
Commonly this is either concatenation or a string template literal (we prefer to use a string template literal). 
For an example where a string template literal is needed, see https://github.com/specify/specify7/blob/7324c28adc552ecb2aefd4c0533433001103cffc/specifyweb/frontend/js_src/lib/components/AppResources/Aside.tsx#L72-L77

```suggestion
      <H3 className="flex items-center gap-2">
```

If configured, ESLint would yell at you for this! 
Though this suggested change should occur automatically when the `Lint Code with ESLint + Prettier` commit is pushed.

Reply:
That's what we pay ESLint to do!!! /s

Reply:
@grantfitzsimmons you marked these comments as resolved, but I don't see them resolved in the code - did you forget to push or are you still working on them?

Reply:
Should occur with 'Lint Code with ESLint + Prettier', no?

Reply:
Yes, this problem is auto-fixable so the `ESLint + Prettier` commit should automatically resolve these. 
Though worryingly, the commit never ran even though it theoretically _should of_  after https://github.com/specify/specify7/pull/4474/commits/9809a657f3ea000aacf6d5ffe7ca563dc5046802 was pushed: the frontend tests were ran and succeeded.